netbox/netbox/views/generic/bulk_views.py

```diff
@@ -541,6 +541,9 @@ class BulkEditView(GetReturnURLMixin, BaseMultiObjectView):
     def get_required_permission(self):
         return get_permission_for_model(self.queryset.model, 'change')
 
+    def extra_object_field_operations(self, form, obj):
```

Comment:
Can you add a doc string as to what this is for, for future reference.

Reply:
Good call; done.